tests/acceptance/features/bootstrap/WebDavLockingContext.php

```diff
@@ -690,7 +699,11 @@ public function unlockItemAsPublicWithLastLockOfUserAndItemUsingWebDavAPI(
 	 * @return void
 	 */
 	public function unlockItemAsPublicUsingWebDavAPI(string $itemToUnlock) {
-		$user = $this->featureContext->getLastCreatedPublicShareToken();
+		if ($this->featureContext->isUsingSharingNG()) {
+			$user = $this->featureContext->shareNgGetLastCreatedLinkShareToken();
+		} else {
+			$user = $this->featureContext->getLastCreatedPublicShareToken();
+		}
```

Comment:
```suggestion
		$user = ($this->featureContext->isUsingSharingNG()) ? $this->featureContext->shareNgGetLastCreatedLinkShareToken() : $this->featureContext->getLastCreatedPublicShareToken();
```
`$user` or `$token`?